### ISSUE TITLE
Allow config value re-ordering for lists via drag and drop

### ISSUE THREAD

**Issue by Abrynos** [JustArchiNET/ASF-ui#1458]
## Purpose
For some configuration properties such as `FarmingOrders` the order of selection matters. If I want to change the order of the currently selected values, currently I have to remove a value from the middle and add it again (placing it at the end).

## Solution
Implement drag-and-drop for the selected values, which are shown below the dropdown menu.

## Alternatives
Don't do anything and leave it as is.

--- 

Thank you in advance for considering this enhancement.

**Comment by MrBurrBurr** [comment]
Good catch, this will be a nice enhancement for the user. The current flow for changing farming order is very user-unfriendly.

I just wonder if there is a sane way to implement this without using a library. Lets see when the time comes to implement this.
